example/lib/test_scenarios/sensor_tag_test_scenario.dart

```diff
@@ -74,7 +75,17 @@ class SensorTagTestScenario {
     CharacteristicWithValue readValue = await peripheral.readCharacteristic(
         SensorTagTemperatureUuids.temperatureService,
         SensorTagTemperatureUuids.temperatureDataCharacteristic);
-    log("Reada temperature value ${readValue.value}");
+    log("Read temperature value ${readValue.value}");
+
+    log("Start monitoring temperature");
```

Comment:
Move monitoring before the first write and change name of the method to `_readWriteMonitor`